yb-voyager/src/query/queryparser/ddl_processor.go

```diff
@@ -354,6 +375,9 @@ func (t *Table) addConstraint(conType pg_query.ConstrType, columns []string, spe
 	generatedConName := tc.generateConstraintName(t.TableName)
 	conName := lo.Ternary(specifiedConName == "", generatedConName, specifiedConName)
 	tc.ConstraintName = conName
+	if conType == FOREIGN_CONSTR_TYPE {
```

Comment:
why doing this in case of foreign key only. Can this be a unconditional step?

Reply:
reference table is only present in case of foreign keys so using the field only in that case as it will be nil in other cases